docs/standard/security/cross-platform-cryptography.md

```diff
@@ -0,0 +1,361 @@
+
+#### The Disallowed store
+
+| Scenario                                    | Windows | Linux | macOS           |
+|---------------------------------------------|---------|-------|-----------------|
+| Open CurrentUser\Disallowed (ReadOnly)      | ✔️     | ⚠️    | ✔️             |
+| Open CurrentUser\Disallowed (ReadWrite)     | ✔️     | ⚠️    | ❌              |
+| Open CurrentUser\Disallowed (ExistingOnly)  | ✔️     | ⚠️    | ✔️ (if ReadOnly) |
+| Open LocalMachine\Disallowed (ReadOnly)     | ✔️     | ❌    | ✔️             |
+| Open LocalMachine\Disallowed (ReadWrite)    | ✔️     | ❌    | ❌              |
+| Open LocalMachine\Disallowed (ExistingOnly) | ✔️     | ❌    | ✔️ (if ReadOnly) |
+
+On Linux, the `Disallowed` store is not used in chain building, and attempting to add contents to it results in a <xref:System.Security.Cryptography.CryptographicException>. A <xref:System.Security.Cryptography.CryptographicException> is thrown when opening the `Disallowed` store if it has already acquired contents.
+
+On macOS, the CurrentUser\Disallowed and LocalMachine\Disallowed stores are interpretations of the appropriate SecTrustSettings results for certificates whose trust is set to `Always Deny`.
+
+#### Nonexistent store
+
+| Scenario                                         | Windows | Linux | macOS |
+|--------------------------------------------------|---------|-------|-------|
+| Open non-existent store (ExistingOnly)           | ❌     | ❌     | ❌    |
+| Open CurrentUser non-existent store (ReadWrite)  | ✔️     | ✔️     | ⚠️   |
+| Open LocalMachine non-existent store (ReadWrite) | ✔️     | ❌     | ❌    |
+
+On macOS, custom store creation with the X509Store API is supported only for `CurrentUser` location. It will create a new keychain with no password in the user's keychain directory (*~/Library/Keychains*). To create a keychain with password, a P/Invoke to `SecKeychainCreate` could be used. Similarly, `SecKeychainOpen` could be used to open keychains in different locations. The resulting `IntPtr` can be passed to [`new X509Store(IntPtr)`](xref:System.Security.Cryptography.X509Certificates.X509Store.%23ctor(System.IntPtr)) to obtain a read/write-capable store, subject to the current user's permissions.
+
+### X509Chain
+
+macOS doesn't support Offline CRL utilization, so `X509RevocationMode.Offline` is treated as `X509RevocationMode.Online`.
+
+macOS doesn't support a user-initiated timeout on CRL (Certificate Revocation List) / OCSP (Online Certificate Status Protocol) / AIA (Authority Information Access) downloading, so `X509ChainPolicy.UrlRetrievalTimeout` is ignored.
+
+## Additional resources
+
+[.NET Cryptography Model](cryptography-model.md)
+[.NET Cryptographic Services](cryptographic-services.md)
```

Comment:
Are these supposed to render side by side, or as a bulleted list?  (Currently they're SxS)

Reply:
Bulleted list, will update, thanks.